src/main/kotlin/no/skatteetaten/aurora/boober/service/resourceprovisioning/DatabaseSchemaProvisioner.kt

```diff
@@ -266,4 +270,16 @@ class DatabaseSchemaProvisioner(
     data class DbApiEnvelope(val status: String, val items: List<DbhSchema> = listOf())
 
     data class DbhErrorResponse(val status: String, val items: List<String>, val totalCount: Int)
+
+    private fun Exception.getDbhErrorMessage(): String {
+        if (this is RestClientResponseException) {
+            try {
+                return jacksonObjectMapper().readValue<DbhErrorResponse>(this.responseBodyAsString)
+                    .let { it.items.firstOrNull()?.let { msg -> " $msg." } ?: "" }
```

Comment:
trenger du .let her i den første? kan du ikke si
`return jacksonObjectMapper().readValue<DbhErrorResponse>(this.responseBodyAsString)?.items?.firstOrNull()`